source/client-side-encryption/tests/README.rst

```diff
@@ -596,3 +596,66 @@ Data keys created with AWS KMS may specify a custom endpoint to contact (instead
 
    Expect this to fail with an exception with a message containing the string: "parse error"
 
+Bypass spawning mongocryptd
+===========================
+
+The following tests that setting ``mongocryptdBypassSpawn`` really does bypass spawning mongocryptd.
+
+#. Create a MongoClient configured with auto encryption (referred to as ``client_encrypted``)
+
+   Configure the required options. Use the ``local`` KMS provider as follows:
+
+   .. code:: javascript
+
+      { "local": { "key": <base64 decoding of LOCAL_MASTERKEY> } }
+
+   Configure with the ``keyVaultNamespace`` set to ``admin.datakeys``.
+
+   Configure the following ``extraOptions``:
+
+   .. code:: javascript
+
+      {
+        "mongocryptdBypassSpawn": true
+        "mongocryptdSpawnArgs": [ "--pidfilepath=bypass-spawning-mongocryptd.pid", "--port=27021"]
+      }
+
+   Drivers MAY pass a different value to ``--port`` if they expect their testing infrastructure to be using port 27021. Pass a port that should be free.
+
+#. Use ``client_encrypted`` to run a "ping" command.
```

Comment:
Currently, a driver could pass this test even if mongocryptdBypassSpawn=true was mistakenly implemented as bypassAutoEncryption=true. What do you think about validating that auto encryption still works in this case?

Reply:
That does seem like a good idea. Updated the test to check that auto encryption still works (by spawning mongocryptd manually via a separate temporary MongoClient with auto encryption)